docs/v2-system-addon/remote_cfr.md

```diff
@@ -53,3 +53,8 @@ Services.prefs.setBoolPref("browser.newtabpage.activity-stream.asrouter.devtools
 
 **4. Go to `about:newtab#devtools`**
 There should be a "cfr-remote" provider listed.
+
+## Using the staging server for Remote CFR
+
+If your message is published in the staging environment the easiest way to test is using the [Remote Settings Devtools](https://github.com/mozilla/remote-settings-devtools/releases) addon. You can install this by going to `about:debugging` and using the `Load Temporary Addon` feature.
+You have to load the `manifest.json` file of the addon in order to install it. The devtools allow you to switch your profile between production and staging and takes care of correctly flipping all the required preferences.
```

Comment:
Hmm, the way I installed this devtool addon is slightly different than yours. After clicking on the "Load temporary addon" button, then selecting the zip file such as "remote-settings-devtools-1.1.0.zip", that's it for me. Would this work for you?